### DISCUSSION TITLE
Admit card and ID card still not able to register for Octernship

### DISCUSSION THREAD

**Discussion by ashishsah1000** [community/community#53229] [GitHub Education]
I am pursuing my Master of Computer Application from Srinath University, Jharkhand, India. Want to register for Octernship.
As soon I click on apply as a student it takes me to Are you a student (developer pack offer). I availed it in 2017 when I started my bachelor's degree. Right now I have no intent to get the pack, I just want to join the internship program and learn. But even after providing the correct docs my application is rejected. 
I have an examination admit card and my college ID card. Please help me to get into this program here are the images of the admit card and id card.

#52688 #52185

**Comment by ashishsah1000** [comment]
I guess I already got the student pack in 2017 the reason it is being rejected. But being a student I can still participate in Octernship without student pack. Right?

**Reply by samus-aran** [reply]
Hi @ashishsah1000 You need to be a verified student yes, not necessarily to have the Student Developer Pack. 

> To apply for the Octernships program, you must follow the requirements listed below:
> - 18+ years old
> - A verified student on GitHub Global Campus
> - An active contributor on GitHub
> - Enrolled in an institution with work eligibility in one of the ten countries: India, Singapore, Indonesia, Malaysia, Vietnam, Philippines, Thailand, Mexico, Nigeria, and Colombia.
